modules/agent-installer-architectures.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="agent-install-networking_{context}"]
+= Optional: Verifying architectures on the Agent-based installer 
+
+You can check which architecture your {product-title} agent installer cluster is using by viewing the version binary.
+
+.Prerequisites: 
+
+* You installed the OpenShift CLI (oc)
+* You have the openshift-installer binary
+
+.Procedure: 
+
+. Check your release payload by running the following command: 
+[source,terminal]
++
+----
+$ ./openshift-install version
+----
++
+.Example output 
+[source,terminal]
+----
+./openshift-install 4.15.0-ec.3
```

Comment:
missed this on my first pass.. this should be `openshift-install version`

Reply:
Oh wait, I don't think the command needs to be in the example output.

Reply:
ok.. i'm being dumb.. that is part of the output 🤦 
cool if we make it generic? something like
`./openshfit-install 4.15.z` or something?